src/lib/net/bond.ts

```diff
@@ -0,0 +1,11 @@
+import { BigNumber } from '@ethersproject/bignumber'
+
```

Comment:
`Bond` as a file name is a bit confusing to me. Why not calling it `constants`?

Reply:
I see that `bond` might be confusing. `constants` is also fine. I used `bond` since constants is the analogous for `utils`, which I usually against. But we can rename until we find a better name.

About moving to the top level, this is related to the structure discussion we had. I wouldn't move it, maybe the `net` name is the problem.